app/controllers/users_controller.rb

```diff
@@ -11,4 +11,26 @@ def index
     @pagy, @users = pagy(@users,  items: Company::PAGE_SIZE)
     respond_to { |format| format.html }
   end
+
+  def edit
+    respond_to { |format| format.html }
+  end
+
+  def update
+    respond_to do |format|
+      if @user.update(edit_params)
+        flash[:success] = t('flash_messages.update', name: t('shared.user'))
+        format.html { redirect_to user_path(@user) }
+      else
+        flash[:danger] = t('flash_messages.error', error_msg: @user.errors.full_messages.first)
+        format.html { render 'edit' }
+      end
```

Comment:
Move logic outside the respond_to block. To handle different outputs based on successful update, you can use a variable or check the value of flash success.

Reply:
done